packages/server/utils/isRequestToJoinDomainAllowed.ts

```diff
@@ -0,0 +1,37 @@
+import isCompanyDomain from './isCompanyDomain'
+import getRethink from '../database/rethinkDriver'
+
+const hasEligibleOrg = async (activeDomain: string) => {
+  const r = await getRethink()
+  return r
+    .table('Organization')
+    .getAll(activeDomain, {index: 'activeDomain'})
+    .filter((org) => org('featureFlags').contains('promptToJoinOrg'))
+    .filter((org) => org('tier').ne('enterprise'))
```

Comment:
+1 could use `.and` here so we use one filter method instead of three, but it might be less readable

Reply:
I think rethinkdb should optimize these queries, so if no strong preference, feels like multiple `filters` could be more readable in this case.